### PR TITLE
Modify home read me

### PR DESCRIPTION
**Pull request by MeniZalzman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,46 +24,46 @@ The NuGet Blog is where we announce new features, write engineering blog posts,
...
+  * Visual Studio 2015 Extension
+  * PowerShell CmdLets
 
-* [NuGet (on CodePlex)](http://nuget.codeplex.com) - the NuGet command-line tool, Visual Studio Extension and PowerShell CmdLets
-* [NuGetGallery](https://github.com/NuGet/NuGetGallery) - the current NuGet Gallery
-
-We are working hard to make NuGet a modern package manager for .NET. The repos where the action around that is happening:
+* [NuGet V2](https://github.com/NuGet/NuGet2) - this repo contains the following clients:
+  * NuGet command-line tool 2.9
+  * Visual Studio Extension (Privouse versions e.g. Visual studio 2013)
```
Comment (yishaigalatzer) [inline README.md]: typo: previous

File: README.md [hunk 2]
```diff
@@ -24,46 +24,46 @@ The NuGet Blog is where we announce new features, write engineering blog posts,
...
-- Download the latest version of [nuget.exe](https://dist.nuget.org/win-x86-commandline/latest/nuget.exe), and add its directory to PATH
 
-Steps to build:
-- Start powershell. Create a directory, cd into that directory
-- Run `git clone https://github.com/NuGet/Home.git`
-- Run `Home\clone-repos.ps1`
-- Run `Home\build-nuget.ps1`. The generated vsix will be  NuGet.VisualStudioExtension\src\VsExtension\bin\Debug\NuGet.Tools.vsix.
+###Steps to build the clients tools repo:
+- Clone [NuGet.Client](https://github.com/nuget/nuget.client) Repo by running the following command `git clone https://github.com/NuGet/NuGet.Client`
+- Start powershell. cd into the clone repo directory
```
Comment (yishaigalatzer) [inline README.md]: put each step on a separate line.

CD into the cloned directory

File: README.md [hunk 3]
```diff
@@ -24,46 +24,46 @@ The NuGet Blog is where we announce new features, write engineering blog posts,
...
+- Run `.\build.ps1 -CleanCache`
+
+######In case you have build issues please clean the local repo using `git clean -xdf` and try building
```
Comment (yishaigalatzer) [inline README.md]: and retry building

**Comment by csharpfritz** [comment]
Can we provide links to the Prerequisites? Let's link to visualstudio.com, the Visual Studio SDK page, and the Windows 10 tools... we want to make it real easy to download and get the tools that our contributors need.

Also, can we change the last bullet to read: "Add the directory containing msbuild v14 to the path: " and then list the "set path=" statement to add the sample path to the end of the path?

**Comment by MeniZalzman** [comment]
Added the needed links, good catch about the MSBuild requirement we don't need it anymore

**Pull request merged**